test/e2e/network/dual_stack.go

```diff
@@ -0,0 +1,300 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package network
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
+
+	apps "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2edeploy "k8s.io/kubernetes/test/e2e/framework/deployment"
+	imageutils "k8s.io/kubernetes/test/utils/image"
+)
+
+// Tests for ipv6 dual stack feature
+var _ = SIGDescribe("[Feature:IPv6DualStackAlphaFeature] [LinuxOnly]", func() {
+	f := framework.NewDefaultFramework("dualstack")
+
+	var cs clientset.Interface
+	var podClient *framework.PodClient
+
+	ginkgo.BeforeEach(func() {
+		// This test suite requires aks-engine cluster.
+		//framework.SkipUnlessProviderIs("azure")
```

Comment:
nit: dead code? the [Feature] tag should be sufficient

Reply:
Done